### DISCUSSION TITLE
加载RWKV模型时报错，找不到啥问题，求助

### DISCUSSION THREAD

**Discussion by slimwang1111** [wenda-LLM/wenda#141] [Q&A] [answered]
FileNotFoundError: [Errno 2] No such file or directory: 'model\\RWKV-4-Raven-7B-v9-Eng49-Other1-20230414-ctx4096.pth'
serving on 0.0.0.0:17860 view at http://127.0.0.1:17860
E:\GPT\plugins\textrank4zh\Segmentation.py:37: ResourceWarning: unclosed file <_io.BufferedReader name='E:\\GPT\\plugins\\textrank4zh\\stopwords.txt'>
  for word in codecs.open(self.stop_words_file, 'r', 'utf-8', 'ignore'):
ResourceWarning: Enable tracemalloc to get the object allocation traceback
['bing', '5']
这个是什么问题啊？

**Comment by l15y** [comment, marked as answer]
No such file or directory: 'model\RWKV-4-Raven-7B-v9-Eng49-Other1-20230414-ctx4096.pth'

**Reply by slimwang1111** [reply]
![image](https://user-images.githubusercontent.com/130847892/232234294-2f99fb36-4c33-4d8a-b7f3-d3ee4d7e3257.png)
当然，我的问题肯定是因为文件夹里面有；）

**Reply by slimwang1111** [reply]
![image](https://user-images.githubusercontent.com/130847892/232234540-8d7f7738-bc0d-459a-b58b-2b3823bc1fdb.png)
怎么弄都报这个错，另外两个模型都没问题，RWKY就报这个错误，求教！

**Reply by l15y** [reply]
文件名删了百分号

**Reply by slimwang1111** [reply]
问题解决，非常感谢！
